x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/host_isolation_exceptions_list.test.tsx

```diff
@@ -26,31 +25,25 @@ describe('When on the host isolation exceptions page', () => {
   let render: () => ReturnType<AppContextTestRender['render']>;
   let renderResult: ReturnType<typeof render>;
   let history: AppContextTestRender['history'];
-  let waitForAction: AppContextTestRender['middlewareSpy']['waitForAction'];
   let mockedContext: AppContextTestRender;
 
   const useEndpointPrivilegesMock = useEndpointPrivileges as jest.Mock;
+  const waitForApiCall = () => {
```

Comment:
Could we add a param to this function and try to wait for a specific API call. Now it seems you are waiting for an API call without taking care of which call it is. I think would be interesting to be able to wait for a get list API call, post exception API call, etc. What do you think?